### ISSUE TITLE
Finalize

### ISSUE THREAD

**Issue by tirthbha** [ubsuny/23-Homework4G5#59]
@uarif Let's finalize the documentation. Once your are done, mention here in this issue. 
I think to make changelog is the final part that needs to be finished, right?

**Comment by uarif** [comment]
Yes, I am about to finish the rest of the things, Actually When I merge, it doesn't replace the old documentation in the main repository. It just adds a new documentation file. IDK why, Let me make the changes and add it. We can delete the other documentation files at the end and leave the final one.

**Comment by tirthbha** [comment]
@uarif You can delete the old documentation file but please add the content of that old file to the new one. And the file cannot be replaced because you named the file with big letter i.e Docunentation.md however, old one is documentation.md.
